bevrand.proxyapi/Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM node:11.12-alpine
+FROM node:lts-alpine
 
 RUN mkdir -p /usr/src/app
 WORKDIR /usr/src/app
 
 # Copy package.json first, so dependencies can be installed seperately
-COPY package.json /usr/src/app/package.json
-RUN npm install --silent
+COPY package*.json /usr/src/app/
+RUN npm ci --only=production
 
 COPY . /usr/src/app
 
 EXPOSE 5000
-CMD ["node", "server.js"]
\ No newline at end of file
+CMD ["npm", "start"]
```

Comment:
Is npm start production grade?

Reply:
https://github.com/nodejs/docker-node/blob/master/docs/BestPractices.md#handling-kernel-signals --> nee